spec/linter-eslint-spec.js

```diff
@@ -213,6 +215,44 @@ describe('The eslint provider for Linter', () => {
     })
   })
 
+  describe('when a file is not specified in .eslintignore file', async () => {
+    it('will give warnings when linting the file', async () => {
+      const tempPath = await copyFileToTempDir(path.join(eslintIgnoreDir, 'ignored.js'))
+      const tempDir = path.dirname(tempPath)
+
+      const editor = await atom.workspace.open(tempPath)
+      atom.config.set('linter-eslint.disableEslintIgnore', false)
+      await copyFileToDir(path.join(eslintIgnoreDir, '.eslintrc.yaml'), tempDir)
+
+      const messages = await lint(editor)
+      expect(messages.length).toBe(1)
+      rimraf.sync(tempDir)
+    })
+  })
+
+  describe('when a file is specified in an eslintIgnore key in package.json', () => {
+    it('will still lint the file if .eslintignore file', async () => {
```

Comment:
`will still lint the file if an .eslintignore file is present`

Reply:
👍